lib/Distribution/Sle/AgamaDevel.pm

```diff
@@ -26,6 +31,12 @@ sub get_grub_menu_agama {
     });
 }
 
+sub get_grub_menu_installed_system {
```

Comment:
looks like we are repeating over and over, what about to overwrite only this method but use as parent Leap, if there are not significant differences, we could do that for now, later we will see...

Reply:
Done